bubbles/tl_commands.py

```diff
@@ -26,62 +22,54 @@
 )
 from bubbles.tl_utils import TLJob
 
-# class PeriodicCheck(TLJob):
```

Comment:
I'd like to keep this or refactor it somehow so that it doesn't get removed. It's useful for testing locally.